spec/libtest/test_subject.h

```diff
@@ -373,4 +393,25 @@ void test_subject_raise_file_error2(TestSubject* self, int foo, GError** error);
  */
 TestIface *test_subject_return_myself_as_interface(TestIface  *self);
 
+/**
+ * test_subject_get_crystallize_docs:
+ *
+ * Used to test documentation conversion to Crystal.
+ */
+void test_subject_get_crystallize_docs();
+
+/**
+ * test_subject_set_crystallize_docs:
+ *
+ * Used to test documentation conversion to Crystal.
+ */
+void test_subject_set_crystallize_docs(TestSubject *self, const gchar* data);
+
+/**
+ * test_subject_is_crystallize_docs: (get-property Boolean)
+ *
+ * Used to test documentation conversion to Crystal.
+ */
+void test_subject_is_crystallize_docs(TestSubject *self);
+
```

Comment:
Better to remove these and use the already existing functions mentioned in comments above, otherwise they would need also an empty implementation on test_subject.c to avoid missing symbols in the shared library.